libs/ml/include/ml/distributed_learning/distributed_learning_client.hpp

```diff
@@ -552,6 +551,39 @@ void TrainingClient<TensorType>::Initialise()
   export_buffer_thread_ = std::make_shared<std::thread>(&TrainingClient::ExportBufferLoop, this);
 }
 
+///////////////////////
+/// private methods ///
+///////////////////////
+
+/**
+ * Adds gradient to own gradient queue
+ * @param gradient
+ */
+template <class TensorType>
+void TrainingClient<TensorType>::QueueAddGradient(GradientType &gradient)
```

Comment:
It's good that you renamed this function, but it will be removed in next PR and replaced with iLearnerNetworker.

Reply:
But just leave it like that I will merge it.